### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brandonxxlee**
-Change schema to better reflect how PieCentral was made to do. 
-Change branch naming scheme to project_name/feature_name or project_name/name/feature_name to better organize the branches and find specific branches
-Staff will edit their specific branches or feature branches.
-Rebase onto master to create a linear commit history. To push to master staff must rebase onto master locally, then open a pull request. 
-All code will be code reviewed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,15 +15,41 @@
 
 ### If you want to learn more about these repositories, check out their directories!
 
-### A note to contributers:
```
Comment (nikitakit) [inline README.md:18]: I think this should stay

File: README.md [hunk 2]
```diff
@@ -15,15 +15,41 @@
...
+```
+
+### Creating a Branch
+Naming convention: project_name/feature_name
+
+Feel free to name "feature_name" anything and/or use more slashes.
+
+e.g. dawn/UDPintegration, atalanta/andy/UDPintegration
+```
+$ git branch {project_name/feature_name}
```
Comment (nikitakit) [inline README.md:37]: Change of branch naming convention LGTM, because it can help resolve cross-project branch confusion.

**Review by nikitakit — APPROVED** [review]

**Review by nikitakit — APPROVED** [review]
Trying to approve+merge this...

**Review by onibrow — APPROVED** [review]
mcomment

**Pull request merged**